eng/tools/internal/exports/exports.go

```diff
@@ -54,6 +54,8 @@ type Func struct {
 
 	// a comma-delimited list of the return types
 	Returns *string `json:"returns,omitempty"`
+
```

Comment:
Please add a comment about the param.

Reply:
comment: ”mark long running operation“ is it okay？

Reply:
could you please provide some examples of this string?

Reply:
![image](https://user-images.githubusercontent.com/46921893/203676644-281a31a7-6d92-4f1a-8d82-08422f787ea2.png)